### PR TITLE
ISPN-10009 - If the test configuration failed, the test should fail as well

### PR DESCRIPTION
**Pull request by diegolovison**
In the class `MultipleCacheManagersTest` change the following method
```java
   @BeforeMethod(alwaysRun = true)
   public void createBeforeMethod() throws Throwable {
      assert false;
      if (cleanupAfterMethod()) callCreateCacheManagers();
   }
```

The report output will be
```xml
  <testcase classname="org.infinispan.functional.distribution.rehash.FunctionalTxTest" name="testDoubleIncrementAfterTopology" time="0.0">
    <skipped/>
  </testcase> <!-- testDoubleIncrementAfterTopology -->
  <testcase classname="org.infinispan.functional.distribution.rehash.FunctionalTxTest" name="testReadWriteKeyBeforeTopology" time="0.0">
    <skipped/>
  </testcase> <!-- testReadWriteKeyBeforeTopology -->
  <testcase classname="org.infinispan.functional.distribution.rehash.FunctionalTxTest" name="testReadWriteManyEntriesAfterTopology" time="0.0">
    <skipped/>
  <testcase classname="org.infinispan.functional.distribution.rehash.FunctionalTxTest" name="createBeforeMethod" time="0.001">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.infinispan.test.MultipleCacheManagersTest.createBeforeMethod(MultipleCacheManagersTest.java:131)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 19 stack frames]]>
    </failure>
  </testcase>
```
It is not wrong at all because the test was skipped due a failure in the test configuration BUT it is expensive to analyze all skipped tests to understand if it was failed because of a test configuration or because of a SkipException or because of `@Ignore`

I am proposing here with this PR is that if a test configuration failed then the test should fail as well.
With the fix the report output will be
```xml
  <testcase classname="org.infinispan.functional.distribution.rehash.FunctionalTxTest" name="testDoubleIncrementAfterTopology" time="0.0">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.infinispan.test.MultipleCacheManagersTest.createBeforeMethod(MultipleCacheManagersTest.java:131)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- testDoubleIncrementAfterTopology -->
  <testcase classname="org.infinispan.functional.distribution.rehash.FunctionalTxTest" name="testReadWriteKeyBeforeTopology" time="0.0">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.infinispan.test.MultipleCacheManagersTest.createBeforeMethod(MultipleCacheManagersTest.java:131)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- testReadWriteKeyBeforeTopology -->
  <testcase classname="org.infinispan.functional.distribution.rehash.FunctionalTxTest" name="testReadWriteManyEntriesAfterTopology" time="0.0">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.infinispan.test.MultipleCacheManagersTest.createBeforeMethod(MultipleCacheManagersTest.java:131)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 19 stack frames]]>
    </failure>
  </testcase> 
```
Each test is now failed with the configuration stack trace

### CODE REVIEW

**Comment by diegolovison** [comment]
@danberindei If a pick a random failure I will see
```
Leaked threads: 
  {jgroups-5,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {NioServer.Selector [/127.0.0.1:8400]-2,TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t1: possible sources [UNKNOWN]},
  {MultiHotRodServerNonIndexedQueryTest-Client-Async-87-1: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t2: possible sources [UNKNOWN]},
  {jgroups-11,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t3: possible sources [UNKNOWN]},
  {remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p731-t1: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t5: possible sources [UNKNOWN]},
  {Hibernate Search: async deletion of index segments-1: possible sources [UNKNOWN]},
  {HotRod-TransactionalIndexingTest-ServerMaster-347-1: possible sources [UNKNOWN]},
  {remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p731-t2: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t4: possible sources [UNKNOWN]},
  {remote-thread-TwoCachesSharedIndexTest-NodeA-p683-t3: possible sources [UNKNOWN]},
  {remote-thread-TransactionalIndexingTest-NodeA-p739-t2: possible sources [UNKNOWN]},
  {MultiHotRodServerNonIndexedQueryTest-Client-Async-86-2: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t5: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerHandler-341-1: possible sources [UNKNOWN]},
  {TOA-Delivery-Thread-MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {Controller-remote-thread-TransactionalIndexingTest-NodeA: possible sources [UNKNOWN]},
  {jgroups-4,JBMARRemoteQueryDslConditionsTest-NodeA-52659: possible sources [UNKNOWN]},
  {HotRod-ContinuousQueryObjectStorageTest-ServerHandler-193-1: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerHandler-341-3: possible sources [UNKNOWN]},
  {jgroups-8,MultiHotRodServerNonIndexedQueryTest-NodeB-63824: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerIO-328-1: possible sources [UNKNOWN]},
  {jgroups-11,MultipleIndexedCacheTest-NodeC-54341: possible sources [UNKNOWN]},
  {remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p710-t3: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t4: possible sources [UNKNOWN]},
  {remote-thread-TwoCachesSharedIndexTest-NodeA-p683-t4: possible sources [UNKNOWN]},
  {remote-thread-ContinuousQueryObjectStorageTest-NodeA-p329-t5: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t3: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t6: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t6: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t1: possible sources [UNKNOWN]},
  {Hibernate Search: async deletion of index segments-1: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeA-p676-t5: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerHandler-321-2: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerMaster-343-1: possible sources [UNKNOWN]},
  {timeout-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p732-t1: possible sources [UNKNOWN]},
  {Lucene Merge Thread #2 for index accounts_protobuf: possible sources [UNKNOWN]},
  {MultiHotRodServerNonIndexedQueryTest-Client-Async-86-1: possible sources [UNKNOWN]},
  {HotRod-TransactionalIndexingTest-Scheduled-346-1: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t2: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t3: possible sources [UNKNOWN]},
  {remote-thread-TwoCachesSharedIndexTest-NodeA-p683-t5: possible sources [UNKNOWN]},
  {jgroups-10,MultiHotRodServerNonIndexedQueryTest-NodeB-63824: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerHandler-341-2: possible sources [UNKNOWN]},
  {jgroups-4,TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {RemoteQueryDslConditionsIndexedFieldObjStorageTest-Client-Async-85-1: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-Scheduled-338-1: possible sources [UNKNOWN]},
  {Controller-remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeC: possible sources [UNKNOWN]},
  {jgroups-11,MultiHotRodServerNonIndexedQueryTest-NodeB-63824: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerHandler-321-1: possible sources [UNKNOWN]},
  {jgroups-6,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t1: possible sources [UNKNOWN]},
  {jgroups-3,TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerIO-332-2: possible sources [UNKNOWN]},
  {jgroups-4,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t3: possible sources [UNKNOWN]},
  {Timer runner-1,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerHandler-333-1: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerIO-336-1: possible sources [UNKNOWN]},
  {TwoCachesSharedIndexTest-Client-Async-83-2: possible sources [UNKNOWN]},
  {NioServer.Selector [/127.0.0.1:8202]-2,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-Scheduled-342-1: possible sources [UNKNOWN]},
  {expiration-thread-TransactionalIndexingTest-NodeA-p745-t1: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerIO-340-1: possible sources [UNKNOWN]},
  {TOA-Delivery-Thread-TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {jgroups-8,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t5: possible sources [UNKNOWN]},
  {jgroups-12,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t2: possible sources [UNKNOWN]},
  {jgroups-9,MultiHotRodServerNonIndexedQueryTest-NodeB-63824: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerMaster-339-1: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t4: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeA-p676-t6: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerIO-344-1: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t2: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t1: possible sources [UNKNOWN]},
  {jgroups-3,JBMARRemoteQueryDslConditionsTest-NodeA-52659: possible sources [UNKNOWN]},
  {remote-thread-TransactionalIndexingTest-NodeA-p739-t1: possible sources [UNKNOWN]},
  {remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p731-t3: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerIO-336-2: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t5: possible sources [UNKNOWN]},
  {remote-thread-ContinuousQueryObjectStorageTest-NodeD-p383-t4: possible sources [UNKNOWN]},
  {Timer runner-1,TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {timeout-thread-TransactionalIndexingTest-NodeA-p740-t1: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t6: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerHandler-333-2: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t4: possible sources [UNKNOWN]},
  {jgroups-10,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {expiration-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p736-t1: possible sources [UNKNOWN]},
  {HotRod-ContinuousQueryObjectStorageTest-ServerHandler-161-2: possible sources [UNKNOWN]},
  {jgroups-7,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {jgroups-8,TwoCachesSharedIndexTest-NodeA-16895: possible sources [UNKNOWN]}
Stacktrace
java.lang.AssertionError: 
Leaked threads: 
  {jgroups-5,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {NioServer.Selector [/127.0.0.1:8400]-2,TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t1: possible sources [UNKNOWN]},
  {MultiHotRodServerNonIndexedQueryTest-Client-Async-87-1: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t2: possible sources [UNKNOWN]},
  {jgroups-11,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t3: possible sources [UNKNOWN]},
  {remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p731-t1: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t5: possible sources [UNKNOWN]},
  {Hibernate Search: async deletion of index segments-1: possible sources [UNKNOWN]},
  {HotRod-TransactionalIndexingTest-ServerMaster-347-1: possible sources [UNKNOWN]},
  {remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p731-t2: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t4: possible sources [UNKNOWN]},
  {remote-thread-TwoCachesSharedIndexTest-NodeA-p683-t3: possible sources [UNKNOWN]},
  {remote-thread-TransactionalIndexingTest-NodeA-p739-t2: possible sources [UNKNOWN]},
  {MultiHotRodServerNonIndexedQueryTest-Client-Async-86-2: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t5: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerHandler-341-1: possible sources [UNKNOWN]},
  {TOA-Delivery-Thread-MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {Controller-remote-thread-TransactionalIndexingTest-NodeA: possible sources [UNKNOWN]},
  {jgroups-4,JBMARRemoteQueryDslConditionsTest-NodeA-52659: possible sources [UNKNOWN]},
  {HotRod-ContinuousQueryObjectStorageTest-ServerHandler-193-1: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerHandler-341-3: possible sources [UNKNOWN]},
  {jgroups-8,MultiHotRodServerNonIndexedQueryTest-NodeB-63824: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerIO-328-1: possible sources [UNKNOWN]},
  {jgroups-11,MultipleIndexedCacheTest-NodeC-54341: possible sources [UNKNOWN]},
  {remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p710-t3: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t4: possible sources [UNKNOWN]},
  {remote-thread-TwoCachesSharedIndexTest-NodeA-p683-t4: possible sources [UNKNOWN]},
  {remote-thread-ContinuousQueryObjectStorageTest-NodeA-p329-t5: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t3: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t6: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t6: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t1: possible sources [UNKNOWN]},
  {Hibernate Search: async deletion of index segments-1: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeA-p676-t5: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerHandler-321-2: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerMaster-343-1: possible sources [UNKNOWN]},
  {timeout-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p732-t1: possible sources [UNKNOWN]},
  {Lucene Merge Thread #2 for index accounts_protobuf: possible sources [UNKNOWN]},
  {MultiHotRodServerNonIndexedQueryTest-Client-Async-86-1: possible sources [UNKNOWN]},
  {HotRod-TransactionalIndexingTest-Scheduled-346-1: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t2: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t3: possible sources [UNKNOWN]},
  {remote-thread-TwoCachesSharedIndexTest-NodeA-p683-t5: possible sources [UNKNOWN]},
  {jgroups-10,MultiHotRodServerNonIndexedQueryTest-NodeB-63824: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerHandler-341-2: possible sources [UNKNOWN]},
  {jgroups-4,TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {RemoteQueryDslConditionsIndexedFieldObjStorageTest-Client-Async-85-1: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-Scheduled-338-1: possible sources [UNKNOWN]},
  {Controller-remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeC: possible sources [UNKNOWN]},
  {jgroups-11,MultiHotRodServerNonIndexedQueryTest-NodeB-63824: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerHandler-321-1: possible sources [UNKNOWN]},
  {jgroups-6,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t1: possible sources [UNKNOWN]},
  {jgroups-3,TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerIO-332-2: possible sources [UNKNOWN]},
  {jgroups-4,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t3: possible sources [UNKNOWN]},
  {Timer runner-1,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerHandler-333-1: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerIO-336-1: possible sources [UNKNOWN]},
  {TwoCachesSharedIndexTest-Client-Async-83-2: possible sources [UNKNOWN]},
  {NioServer.Selector [/127.0.0.1:8202]-2,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-Scheduled-342-1: possible sources [UNKNOWN]},
  {expiration-thread-TransactionalIndexingTest-NodeA-p745-t1: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerIO-340-1: possible sources [UNKNOWN]},
  {TOA-Delivery-Thread-TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {jgroups-8,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t5: possible sources [UNKNOWN]},
  {jgroups-12,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t2: possible sources [UNKNOWN]},
  {jgroups-9,MultiHotRodServerNonIndexedQueryTest-NodeB-63824: possible sources [UNKNOWN]},
  {HotRod-RemoteQueryDslConditionsIndexedFieldObjStorageTest-ServerMaster-339-1: possible sources [UNKNOWN]},
  {transport-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p733-t4: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeA-p676-t6: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerIO-344-1: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t2: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t1: possible sources [UNKNOWN]},
  {jgroups-3,JBMARRemoteQueryDslConditionsTest-NodeA-52659: possible sources [UNKNOWN]},
  {remote-thread-TransactionalIndexingTest-NodeA-p739-t1: possible sources [UNKNOWN]},
  {remote-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p731-t3: possible sources [UNKNOWN]},
  {HotRod-MultiHotRodServerNonIndexedQueryTest-ServerIO-336-2: possible sources [UNKNOWN]},
  {notification-thread-MultiHotRodServerNonIndexedQueryTest-NodeB-p709-t5: possible sources [UNKNOWN]},
  {remote-thread-ContinuousQueryObjectStorageTest-NodeD-p383-t4: possible sources [UNKNOWN]},
  {Timer runner-1,TransactionalIndexingTest-NodeA-31303: possible sources [UNKNOWN]},
  {timeout-thread-TransactionalIndexingTest-NodeA-p740-t1: possible sources [UNKNOWN]},
  {transport-thread-TransactionalIndexingTest-NodeA-p741-t6: possible sources [UNKNOWN]},
  {HotRod-TwoCachesSharedIndexTest-ServerHandler-333-2: possible sources [UNKNOWN]},
  {stateTransferExecutor-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p734-t4: possible sources [UNKNOWN]},
  {jgroups-10,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {expiration-thread-MultiHotRodServerNonIndexedQueryTest-NodeC-p736-t1: possible sources [UNKNOWN]},
  {HotRod-ContinuousQueryObjectStorageTest-ServerHandler-161-2: possible sources [UNKNOWN]},
  {jgroups-7,MultiHotRodServerNonIndexedQueryTest-NodeC-62133: possible sources [UNKNOWN]},
  {jgroups-8,TwoCachesSharedIndexTest-NodeA-16895: possible sources [UNKNOWN]}
	at org.infinispan.commons.test.ThreadLeakChecker.performCheck(ThreadLeakChecker.java:196)
	at org.infinispan.commons.test.ThreadLeakChecker.checkForLeaks(ThreadLeakChecker.java:161)
	at org.infinispan.test.AbstractInfinispanTest.testClassFinished(AbstractInfinispanTest.java:144)
	at jdk.internal.reflect.GeneratedMethodAccessor191.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:214)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```
Before rebase with upstream/master let me ask because this can be an unknow issue.

**Comment by danberindei** [comment]
@diegolovison always rebase first :)

If you want to compare with master, there's always https://ci.infinispan.org/job/Infinispan/job/master/

There seems to be something wrong with the test name in the report, failures show as `org.infinispan.cli.interpreter.GrantDenyTest.cli.interpreter.GrantDenyTest`. Whatever is going on there may also affect the thread leak checker.

File: commons/pom.xml [hunk 1]
```diff
@@ -125,11 +125,6 @@
                   <log4j.configurationFile>${log4j.configurationFile}</log4j.configurationFile>
                   <build.directory>${project.build.directory}</build.directory>
                </systemPropertyVariables>
-               <properties combine.self="override">
-                  <usedefaultlisteners>false</usedefaultlisteners>
-                  <listener>${junitListener}</listener>
```
Comment (danberindei) [inline commons/pom.xml]: Why are you removing the listeners?

**Comment by diegolovison** [comment]
@danberindei I am changing my approach. Please don't review this PR.
I will keep it open because I need to run on Jenkins.
Thanks for your patience

**Comment by diegolovison** [comment]
The commit 4b05f8b has
2 failures , 235 skipped and 27,701 tests

**Comment by diegolovison** [comment]
@danberindei it is ready for review

**Comment by galderz** [comment]
@danberindei Is this ready to go in now?

**Comment by danberindei** [comment]
@diegolovison I'm still not convinced this is necessary: if one method fails, I want to see just one failure in the test results. Why would it be better to see 20 failures if the test class has 20 methods?

**Comment by danberindei** [comment]
> It is not wrong at all because the test was skipped due a failure in the test configuration BUT it is expensive to analyze all skipped tests to understand if it was failed because of a test configuration or because of a SkipException or because of @Ignore

Why analyze skipped tests at all? IMO skipped tests should be ignored and only failures should matter.

**Comment by galderz** [comment]
@diegolovison I agree with @danberindei that 1 failure is enough. If there's a problem with that particular failure, raise it up. There's no need for you to do extra arithmetic or analysis.

**Comment by diegolovison** [comment]
Ok. Closing it.

**Pull request closed without merging**